tests/test_dsp.py

```diff
@@ -111,6 +111,18 @@ def test_group_delay_custom_frequencies(impulse_group_delay):
         grp, impulse_group_delay[1][0, frequency_idx], atol=1e-10)
 
 
+@pytest.mark.parametrize("shape", [(4, 1), (1, 4)])
+def test_group_delay_cshape(shape):
+    """Test if group delay function keeps cshape of signals of shape `m x n`
+     with `m = 1` or `n = 1`."""
+    delay = np.random.randint(0, 200, (shape))
+    impulse = pf.signals.impulse(256, delay, np.ones(shape))
```

Comment:
```suggestion
    impulse = pf.signals.impulse(256, 64, np.ones(shape))
```
better would be not to use random inside tests.